packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -65,9 +58,7 @@ export class AttestationService {
     this.config = config;
     this.provider = rpcClient;
     for (const secretKey of secretKeys) {
-      this.secretKeys.push(secretKey);
-      this.publicKeys.push(secretKey.toPublicKey().toBytes());
-      this.validators.push(null);
+      this.validators.set(toHexString(secretKey.toPublicKey().toBytes()), {validator: null, secretKey});
```

Comment:
Doesn't `secretKey.toPublicKey()` have toHex or something method?

Reply:
looks like it does!  will fix this in next commit